libraries/abstractions/transport/secure_sockets/transport_secure_sockets.c

```diff
@@ -145,6 +149,10 @@ int32_t SecureSocketsTransport_Send( NetworkContext_t * pNetworkContext,
 
 /*-----------------------------------------------------------*/
 
+/* MISRA Rule 18.3 flags the following line for not using the const qualifier
+ * on `pNetworkContext`. Indeed, the object pointed by it is not modified
+ * by Secure Sockets, but other implementations of `TransportRecv_t` may do so. */
+/* coverity[misra_c_2012_rule_8_13_violation] */
```

Comment:
In my experience, this does not actually work in suppressing 8.13 violations. It'd be better to remove this message if it doesn't actually work in doing so

Reply:
I would say remove `/* coverity[misra_c_2012_rule_8_13_violation] */`, but the comment can stay. Although, this says 18.3 instead of 8.13.

Reply:
oops~my bad, thanks for the correction.